tests/test_management.py

```diff
@@ -137,6 +137,7 @@ class TestTransmitCourseMetadataManagementCommand(unittest.TestCase, EnterpriseM
     """
     Test the ``transmit_content_metadata`` management command.
     """
+    # pylint: disable=line-too-long
```

Comment:
^ = the strategy for being able to disable the line too long complaints everywhere in this file instead of it being interpreted bizarrely. (This is a known [open] bug with pylint)

Reply:
interesting, this means we can remove this someday once we upgrade pylint I presume

Reply:
Not really no. The original usage was just specifying this on each line where it applied. However, that doesn't work because of the bug for some reason. 

I think enabling line-too-long at all is debatable, but if it's going to be enabled by the rules, currently anything long added to this file will be ignored, which may or may not be desired. 

To put it simply: instead of being able to specify the disable commands for the only 2 lines affected, I had to apply it to the entire class because of the pylint bug.